src/modules/organization/createOrg/create-org-form/CreateOrgForm.vue

```diff
@@ -0,0 +1,339 @@
+<template>
+  <form
+    @submit.prevent="submitForm"
+    class="flex flex-col pt-6 pb-3 px-8 w-full rounded-2xl bg-white"
+  >
+    <h1 class="font-heading text-xl mb-3">General Information</h1>
+    <section class="flex mb-3">
+      <div
+        class="flex justify-center items-center w-8 h-8 border-2 border-primary bg-primary-2 rounded-lg overflow-hidden mr-4"
+      >
+        <base-icon
+          v-if="!fileLoaded"
+          :width="18"
+          :height="18"
+          class="text-primary"
+          ><ImageGalleryIcon
+        /></base-icon>
+        <img
+          v-else
+          :src="uploadedImg"
+          alt="uploadedImg"
+          class="object-cover w-full h-full"
+        />
+      </div>
+      <div class="flex flex-col min-h-full justify-between">

```

Comment:
Profile picture should be separated to new component to use with create user modal

Reply:
Design is kind of very different, so I am not sure how much use this will be.